.DEREK.yml

```diff
@@ -7,6 +7,7 @@ curators:
   - ehazlett
   - fntlnz
   - gianarb
+  - kolyshkin
```

Comment:
Might want to add yourself to the "curators" list as well (like https://github.com/moby/moby/commit/46c713ab5ad20b97de01537a8092d2e747f9ad0e)

Reply:
yep, done